client/app/components/onboardingGuide/GuideCoverPhotoPage.jsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import scss from './styles.scss';
+
+const GuideCoverPhotoPage = (props) => {
+
+  const handleClick = function(e, path) {
+    e.preventDefault();
+    props.changePage(path);
+  }
+
+  return (
+    <div className="container">
+      <a className={ scss.backLink }
+        onClick={ (e) => handleClick(e, "") }
+        href={ props.initialPath } >
+        ‹ { props.t('back_to_todo') }
+      </a>
+
+      <h2 className={ scss.title } >{ props.t('title') }</h2>
+
+      <p className={ scss.description } >
+        { props.t('description') }
+      </p>
+
+      <div className={ scss.sloganImageContainer } >
+        <img src={ props.pageData.infoImage } className={ scss.sloganImage } />
```

Comment:
Is this an image with some slogan text? In any case, there should probably be an alt text here.